content/GOVERNANCE.md

```diff
@@ -0,0 +1,139 @@
+include but is not limited to code, code review, infrastructure work,
+mailing list and chat participation, community help/building,
+education and outreach, design work, etc. We deliberately do not
+set arbitrary quantitative metrics to avoid encouraging behavior
+that plays to the metrics rather than the project's overall well-being.
+We want to encourage a diverse array of backgrounds, viewpoints and
+talents in our team, which is why we explicitly do not define code as
+the sole metric on which Council membership will be evaluated.
+
+If a Council Member becomes inactive in the project for a period of
+one year, they will be considered for removal from the Council. Before
+removal, the inactive Member will be approached by another Council
+member to ask if they plan on returning to active participation. If
+not they will be removed immediately upon a Council vote. If they plan
+on returning to active participation soon, they will be given a grace
+period of one year. If they do not return to active participation
+within that time period they will be removed by vote of the Council
+without further grace period. All former Council members can be
+considered for membership again at any time in the future, like any
+other Project Contributor. Retired Council members will be listed on
+the project website, acknowledging the period during which they were
+active in the Council.
+
+The Council reserves the right to eject current Members if they are
+deemed to be actively harmful to the Project's well-being, and
+attempts at communication and conflict resolution have failed.
+
+### Fiscal Decisions
+
+All fiscal decisions are made by the steering council to ensure any
+funds are spent in a manner that furthers the mission of the Project.
+Fiscal decisions require majority approval by acting steering council
+members.
+
+### Advisory Committee
+
+The Project will consider setting up an Advisory Committee that works to ensure the long-term
+well-being of the Project. The role of the Committee will be to advise the Steering Council.
+
+### Conflict of interest
+
+It is expected that Steering Council and Advisory Committee Members
+will be employed at a wide range of companies, universities and non-profit
+organizations. Because of this, it is possible that Members will have
+conflicts of interest. Such conflicts of interest include, but are not
+limited to:
+
+- Financial interests, such as investments, employment or contracting
+  work, outside of the Project that may influence their work on the
+  Project.
+- Access to proprietary information of their employer that could
+  potentially leak into their work with the Project.
+
+All members of the Council and Committee shall disclose any conflict of
+interest they may have. Members with a conflict of interest in a
+particular issue may participate in Council discussions on that issue,
+but must recuse themselves from voting on the issue.
```

Comment:
You can ignore the false warning here using CLI args in `pre-commit-config.yaml`. Something like this should work -
```yaml
- repo: https://github.com/codespell-project/codespell
  rev: v2.2.5
  hooks:
  - id: codespell
    args: ["-Lrecuse"]
```